springfield/firefox/templates/firefox/releases/nightly-notes.html

```diff
@@ -42,10 +42,7 @@
 
 {% block extra_draft_description %}
   <p class="c-release-draft-text">
-    {%- with twitter='https://twitter.com/FirefoxNightly' -%}
-      ProTip: Be the first to know the latest changes by subscribing to the Nightly <a href="{{ notes_feed }}">Notes
-      Feed</a> and <a href="{{ blog_feed }}">News Feed</a>. Also, don’t forget to follow us on
-      <a href="{{ twitter }}">X (formerly Twitter)</a>!
-    {%- endwith -%}
+    ProTip: Be the first to know the latest changes by subscribing to the Nightly <a href="{{ notes_feed }}">Notes
+    Feed</a> and <a href="{{ blog_feed }}">News Feed</a>.
```

Comment:
Pascal has provided both Mastodon and Bsky links for Nightly, this will probably be cherry-picked from final [bedrock#15995](https://github.com/mozilla/bedrock/pull/15995/files#diff-66dbf4944e2d211b2209fbc9bdb851fc31910d8e0edabba335e0ad711c23dc8b) to match, right?